docs/interface.md

```diff
@@ -21,6 +21,9 @@ The `analysis.json` file should be structured as followed:
     {
       "comment": "ruby.general.some_paramaterised_message",
       "params": { "foo": "param1", "bar": "param2" }
+    },{
```

Comment:
```suggestion
    },
    {
```

Or normalise the others to look like this ;)

Reply:
Good catch. Resolved.

Note: this PR should now be squashed.